test/server/cards/01_SOR/units/EzraBridger.spec.ts

```diff
@@ -0,0 +1,87 @@
+describe('Ezra Bridger', function() {
+    integration(function(contextRef) {
+        describe('Ezra Bridger\'s ability', function() {
+            beforeEach(function () {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        hand: ['moisture-farmer'],
+                        groundArena: ['ezra-bridger#resourceful-troublemaker'],
+                        deck: ['moment-of-peace',
+                            'wampa',
+                            'atst',
+                            'atst',
+                            'atst',
+                            'atst',
+                            'atst',
+                        ]
+                    },
+                    player2: {
+                        base: 'dagobah-swamp',
+                        groundArena: ['death-trooper']
+                    }
+                });
+            });
+
+            it('should trigger when he completes an attack', function () {
+                const { context } = contextRef;
+                const reset = () => {
```

Comment:
I've been encouraging people to do them for now, it makes the long multi-case tests cleaner. We're going to have do a massive refactor anyway if we ever actually do get to the point of having a better test harness, and I don't think having or not having these will have a real impact on the amount of work.

If anything they would make it easier to refactor since we can tell right away which lines are just state reset and aren't part of a specific case.